src/Business Foundation/App/NoSeries/src/Setup/NoSeriesSetupImpl.Codeunit.al

```diff
@@ -137,6 +136,22 @@ codeunit 305 "No. Series - Setup Impl."
         exit(NoSeriesSingle.MayProduceGaps());
     end;
 
+    procedure SetNoSeriesLineFilters(var NoSeriesLine: Record "No. Series Line"; NoSeriesCode: Code[20]; StartingDate: Date)
+    var
+        NoSeries: Codeunit "No. Series";
+        NoSeriesLine2: Record "No. Series Line";
+    begin
+        NoSeriesLine2.SetCurrentKey("Series Code", "Starting Date");
+        NoSeriesLine2.SetRange("Starting Date", 0D, StartingDate);
+        NoSeriesLine2.SetRange("Series Code", NoSeriesCode);
+        NoSeries.OnSetNoSeriesLineFilters(NoSeriesLine2);
+        If NoSeriesLine2.GetFilter("Series Code") <> NoSeriesCode then
+            Error(CodeFieldChangedErr, NoSeriesCode, NoSeriesLine2.GetFilter("Series Code")); // Extensions should never change the code field range, this is a bug that developers should know immediately.
```

Comment:
I believe the comment can be removed, as the content of `CodeFieldChangedErr` actually explains it quite well.
```suggestion
            Error(CodeFieldChangedErr, NoSeriesCode, NoSeriesLine2.GetFilter("Series Code"));
```